OVERVIEW.md

```diff
@@ -313,7 +313,7 @@ JavaScript extracting from files that doesn't match to [fileExtensions](#fileExt
 
 #### Default
 
-Default value is `["**/*.+(htm|html|xhtml)"]`. So JavaScript is extracted from files with `.htm`, `.html` or `.xhtml` extension.
+JavaScript is extracted from files with `.htm`, `.html` or `.xhtml` extension by default. If an array is passed those extensions are concatenated.
```

Comment:
It's not correct. By default it doesn't extract anything.
Array contains file masks, not extensions.
`true` value is use default html mask.